lib/widgets/message_list.dart

```diff
@@ -926,6 +928,10 @@ class MessageWithPossibleSender extends StatelessWidget {
                       ).merge(weightVariableTextStyle(context, wght: 600,
                                 wghtIfPlatformRequestsBold: 900)),
                       overflow: TextOverflow.ellipsis)),
+                  if (messageSender?.isBot ?? false) ...[
+                    const SizedBox(width: 8),
+                    const Icon(ZulipIcons.bot, size: 18, color: Colors.grey),
```

Comment:
Let's match the color to what the web app uses.

I'm also curious how the 18px size is chosen, and the 8px distance from the user's name. To my eye I think those values might both want to be a bit smaller. What does the web app do?

Reply:
> This should be fixed. Even on the webapp the bot icon isn't super prominent, and I have no idea why it's blurred.

I saw [this](https://chat.zulip.org/#narrow/stream/48-mobile/topic/no.20marker.20for.20bots/near/1587470) comment on CZO, so I decided to go with a more prominent icon. Any recommendations from your side are most welcome!

Reply:
On the web, the icon size is 12px, the color is rgb(159, 173, 173), and the distance to the user's name is 5px. 
With the same properties applied, it looks like this:

<img src="https://github.com/zulip/zulip-flutter/assets/59946442/45745acc-a66e-484d-aeea-b607e39ef30a" width="300" />

Should I go with this?

Reply:
Hmm. To me, that all looks good except the size; I think it looks too small now. The font size in the web app is smaller than in the mobile app, so maybe this icon should be scaled up according to that. Maybe try 15px?

Reply:
Applied the changes. Updated the description for the new images. Please have a look!